src/main/java/uk/gov/digital/ho/hocs/notify/api/NotifyService.java

```diff
@@ -82,13 +82,9 @@ public void sendUserAssignChangeEmail(UUID caseUUID, UUID stageUUID, String case
             if (newUserUUID != null) {
                 if (currentUserUUID != null && !newUserUUID.equals(currentUserUUID)) {
                     sendUnAllocateUserEmail(caseUUID, stageUUID, currentUserUUID, caseReference);
-                    if (!newUserUUID.equals(requestData.userIdUUID())) {
-                        sendAllocateUserEmail(caseUUID, stageUUID, newUserUUID, caseReference);
-                    }
-                } else {
-                    if (!newUserUUID.equals(requestData.userIdUUID())) {
-                        sendAllocateUserEmail(caseUUID, stageUUID, newUserUUID, caseReference);
-                    }
+                }
+                if (!newUserUUID.toString().equals(requestData.userId())) {
+                    sendAllocateUserEmail(caseUUID, stageUUID, newUserUUID, caseReference);
```

Comment:
Under this circumstance we don't send an email if the user allocates to themselves, I think it would be better to send the email no matter on who has allocated what to who?

Reply:
I've maintained the existing behaviour, happy to take it to the PM but its outside of this ticket.